### ISSUE TITLE
User can’t scroll on all Settings screens

### ISSUE THREAD

**Issue by ximenabb** [digidem/mapeo-mobile#944]
**Description :** https://www.notion.so/digidem/User-can-t-scroll-on-all-Settings-screens-or-Sync-and-GPS-details-screens-780d392a66544ca38ef637577872dda2
Logged here - 
When text size is increased for accessibility needs or Mm is in other languages with longer description strings (Spanish, Portuguese, etc.), user cannot scroll to view or activate options on lower half of screen. User cannot fully ready details for troubleshooting.

**Todo:**
Have device languages and accessibility settings not negatively impact function of any Mapeo screens.

Priority:
Settings screens are top priority (@marcartro  flagged it as urgent)
]Sync and GPS details can follow if need be.
